### ISSUE TITLE
swag init with `--instanceName` generates invalid variable

### ISSUE THREAD

**Issue by Syuparn** [swaggo/swag#1133]
**Describe the bug**

When `swag init` is executed with `--instanceName`, it generates invalid variable and compile error is raised

**To Reproduce**

1. clone this repository
2. run `swag init --instanceName foo` in sample project `example/basic/`

**Expected behavior**

swag generates variable `SwaggerInfo_foo` .

**Screenshots**

Complie error is raised because SwaggerInfo variable is split into 2 tokens as below

```go
// SwaggerInfo_foo holds exported Swagger Info so clients can modify it
var SwaggerInfo _foo = &swag.Spec{
	Version:          "1.0",
	Host:             "petstore.swagger.io",
	BasePath:         "/v2",
	Schemes:          []string{},
	Title:            "Swagger Example API",
	Description:      "This is a sample server Petstore server.",
	InfoInstanceName: "foo",
	SwaggerTemplate:  docTemplate_foo,
}
```

**Your swag version**
e.g. v1.7.9-p1

**Your go version**
e.g. 1.17.6

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

**Comment by ubogdan** [comment]
I believe this can be closed since it was released with v1.8.0.

**Comment by Syuparn** [comment]
I tried `swag init --instanceName foo` in v1.8.0 and it generates a valid variable `var SwaggerInfofoo`. Thank you so much!
